Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -574,22 +574,26 @@ ExplicitBitVect *wrapRDKFingerprintMol(
     unsigned int fpSize, unsigned int nBitsPerHash, bool useHs,
     double tgtDensity, unsigned int minSize, bool branchedPaths,
     bool useBondOrder, python::object atomInvariants, python::object fromAtoms,
-    python::object atomBits) {
+    python::object atomBits, python::object bitInfo) {
   rdk_auto_ptr<std::vector<unsigned int> > lAtomInvariants =
```

Comment:
Looks like bad things might happen if mol.GetNumAtoms() == 0 unless there is an internal invariant check in the C++ code.  Specifically thinking about lfromAtoms.get() in RDKFingerprintMol